src/app/subnets/components/ReservedRanges/ReservedRanges.tsx

```diff
@@ -175,47 +74,21 @@ const ReservedRanges = ({
 
   useFetchActions([ipRangeActions.fetch]);
 
-  const headers = [
-    {
-      content: Labels.StartIP,
-      className: "start-ip-col",
-      sortKey: "start_ip",
-    },
-    {
-      content: Labels.EndIp,
-      className: "end-ip-col",
-      sortKey: "end_ip",
-    },
-    {
-      content: Labels.Owner,
-      className: "owner-col",
-      sortKey: "owner",
-    },
-    {
-      content: Labels.Type,
-      className: "type-col",
-      sortKey: "type",
-    },
-    {
-      content: Labels.Comment,
-      className: "comment-col",
-      sortKey: "comment",
-    },
-    {
-      content: Labels.Actions,
-      className: "actions-col u-align--right",
-    },
-  ];
-
-  if (showSubnetColumn) {
-    headers.unshift({
-      content: Labels.Subnet,
-      className: "subnet-col",
-      sortKey: "subnet",
-    });
-  }
+  const columns = useReservedRangesColumns(
+    showSubnetColumn,
+    setSidePanelContent
+  );
```

Comment:
Generally prefer not passing around contexts, better to call where it is used. Less props the better.
```suggestion
  const columns = useReservedRangesColumns(
    showSubnetColumn
  );
```